Test

LincolnRuwambara opened the pull request:
Firstly i made changes to the Read Me file, where i added the database schema and more description to the methods that are going to be used in this project.
In the pull request is an SQL file that i exported from PostgreSQL   that will serve the use of creating a database, to gether with an image with the database schema

LincolnRuwambara committed:
Changes have been made on the README file to describe more about the
project. Data base schema was added and methods that are going to be
used when logging in as an Admin or customer

LincolnRuwambara committed:
added an sql file to create the project's database and an image with the
database schema

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+
+
+Goal
+The goal of this project is to create the backend setup of a financial institution (Bank) application.
```
What tasks does your backend solve?

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+
+
+
+Transactions:
```
It's table for one customer or for storage here information about all customers?

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+Application Use:
+  
+Log in Admin Account
+/Admin/admin login
```
It's URL? Like http://localhost:3000/Admin/admin or what?

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+
+If user is already registered
+{
+"status": 400,
```
for this situations better use 409 code

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+"Account Details": [{
+{
+"Account id": 3,
+"Amount": "100 000",
```
Question about amount. In which table you keep information about this? U write that wanna get information about customers. But from which field? And from which table?

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+Log in Customer account
+/customer/customer Login
+METHOD: POST
+status code: 201
```
What the difference between Login and registry? And why u send "Created"(status code 201)?

ilyaKarachun commented on README.md:
```diff
@@ -1,19 +1,233 @@
+Customer:
+This table stores information about the bank�s customer. Contains essential information about the customer used to link the customer to their bank accounts and other services like loans. 
+Account:
+Used to store customer�s account details. Show the date the account was created or the account was delete and relevant information linking to the customer like customer id
```
Plz, add in each tables information about references and their types

LincolnRuwambara committed:
added more information to describe about what the backend application
 is trying to archieve. I also alterd the database schema and removed
 unnecessary tables adding a new table (Currencies ) which is more
 usefull. I also chance the URL links of the admin registration and
 creation and updated the codes recieved

LincolnRuwambara commented:
apologies i re opened the pull request

LincolnRuwambara committed:
made changes to the readme file and explained more about the project,
what is is trying to archiev and the use.  Also changed API structures
that had ambiguas terms

LincolnRuwambara committed:
djusted the md file to look better

LincolnRuwambara committed:
utlook ajustments

LincolnRuwambara committed:
The commit contains essential .js files for my Bank API, also the database connection to POSTGRES

LincolnRuwambara committed:
Merge branch 'test' of https://github.com/LincolnRuwambara/bank_project into test

LincolnRuwambara committed:
resolved the postgres connection error by adding  pool.connect()

LincolnRuwambara committed:
Readme File fix

LincolnRuwambara committed:
Merge branch 'test' of https://github.com/LincolnRuwambara/bank_project into test

LincolnRuwambara committed:
Added more APIs to the project to add,delete and update the customer table
# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# On branch test
# Your branch is ahead of 'origin/test' by 2 commits.
#   (use "git push" to publish your local commits)
#
# Changes to be committed:
#	modified:   README.md
#	modified:   server.js
#	modified:   src/customer/controller.js
#	modified:   src/customer/queries.js
#	modified:   src/customer/routes.js
#

LincolnRuwambara committed:
fixes on the route file

LincolnRuwambara committed:
created a docker file that will containerize the application

LincolnRuwambara committed:
# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# On branch test
# Your branch is up to date with 'origin/test'.
#
# Changes to be committed:
#	modified:   Dockerfile
#	modified:   package.json
#
# Changes not staged for commit:
#	deleted:    dockerfile
#
# Untracked files:
#	.dockerignore
#	.gitignore
#

LincolnRuwambara committed:
added a docker and git ignore file to my project
# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# On branch test
# Your branch is up to date with 'origin/test'.
#
# Changes to be committed:
#	new file:   .dockerignore
#	new file:   .gitignore
#
# Changes not staged for commit:
#	deleted:    dockerfile
#

LincolnRuwambara committed:
remove gitignore files

LincolnRuwambara committed:
removing the node_modules
